internal/handlers/reg_key.go

```diff
@@ -48,49 +49,157 @@ func (api *API) CreateRegKey(c *gin.Context) {
 		return
 	}
 
-	// The vpc has to be owned by the user.
-	var vpc models.VPC
-	db := api.db.WithContext(ctx)
-	if res := api.VPCIsOwnedByCurrentUser(c, db).
-		First(&vpc, "id = ?", request.VpcID.String()); res.Error != nil {
-		c.JSON(http.StatusNotFound, models.NewNotFoundError("vpc"))
+	// use a wg private key as the token, since it should be hard to guess.
+	token, err := wgtypes.GeneratePrivateKey()
```

Comment:
Unrelated to this PR, but I wonder if this is more expensive to generate than a UUID.

Reply:
It likely is, but I would assume it's more random too which is highly desired.

Reply:
looked it up: 
> Randomly generated UUIDs have 122 random bits.  
so that's about 16 bytes vs 32 bytes for wgkeys.